ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -285,9 +315,7 @@ def verify_operator_succeeded(
             )
             raise TimeoutExpiredError
 
-    def search_operator_installed_operators_page(
-        self, operator="OpenShift Container Storage"
-    ):
+    def search_operator_installed_operators_page(self, operator=OCS_OPERATOR):
```

Comment:
For 4.9, operator=ODF_OPERATOR should be used, isn't it?

Reply:
This is the default arguments of method